algorithms/removeExtra.js

```diff
@@ -0,0 +1,8 @@
+const removeExtra =(inp, i=0, prev='', res='')=>{
+  if(i===inp.length){return res};
+  if(inp[i]!=prev){
+    res = res+inp[i]; 
+  }
+return removeExtra(inp, i+1, inp[i], res);
+}
+console.log(removeExtra('helllo'));
```

Comment:
this is awesome, my code is almost 20 lines.

Reply:
thanks Madhu!